build-system/amp.extern.js

```diff
@@ -60,6 +60,20 @@ window.AMP_CONFIG.errorReportingUrl;
 
 window.AMP_CONTEXT_DATA;
 
+// AMP-Analytics Cross-domain iframes
+/** @enum {string} */
+const AMP_ANALYTICS_3P_MESSAGE_TYPE = {
```

Comment:
Since you already include this in a common util class, I don't believe you need it in externs as the compiler will have enforced type check